lib/fake_stripe/stub_stripe_js.rb

```diff
@@ -27,7 +27,16 @@ class StubStripeJS < Sinatra::Base
 
       content_type "text/javascript"
       status 200
-      IO.read(file_path)
+      js = IO.read(file_path)
+      if defined?(@@token)
+        js.gsub(/tok_123/, @@token)
```

Comment:
I think with my suggestion the conditional would go away and this line would become:

```suggestion
        js.gsub(/tok_123/, FakeStripe.js_v3_token)
```

(I have no idea if `js_v3_token` is the correct name for this - it has been a little while since I worked with any of this.)